otlp/traces.go

```diff
@@ -125,6 +120,21 @@ func TranslateTraceRequest(request *collectorTrace.ExportTraceServiceRequest, ri
 					}
 					if isError {
 						attrs["error"] = true
+
```

Comment:
The [semantic conventions for exceptions](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/exceptions/) say that the `name` field must be set to `exception`. I feel like testing that should be part of this. We set `isError` but don't look at that value -- and arguably, maybe we shouldn't do this if it's not leveraging that convention.

Reply:
I've added this condition -- we only do this now if the span event name is "exception" (note that it's no longer doing it on everything that's named as an "error").